docs/src/less/index.less

```
@@ -78,13 +78,17 @@ figure.highlight {
 }
 
 .modules-expander {
-    background: var(--color-neutral-2);
+    background: var(--color-loading-fill-elevated);
```

Did you mean to use `--color-background-elevated` because `loading` variables are for loading components?
```suggestion
    background: var(--color-background-elevated);
```

---

@saiponnada , wow, I had thought I used `--color-background-elevated`. No idea how that happened. LOL. Great catch!

---

@saiponnada , ok, I moved the token to the docs side since we don't have anything semantically comparable in Skin and this is unique to docs. I also made the menu absolute so there's no page shifting on toggle.